modules/update-service-configure-cvo.adoc

```diff
@@ -33,10 +33,16 @@ $ NAME=service
 $ POLICY_ENGINE_GRAPH_URI="$(oc -n "${NAMESPACE}" get -o jsonpath='{.status.policyEngineURI}/api/upgrades_info/v1/graph{"\n"}' updateservice "${NAME}")"
 ----
 
-. Patch the CVO ClusterVersion to use the local OpenShift Update Service:
+. Set the patch for the pull graph data:
 +
 [source,terminal]
 ----
 $ PATCH="{\"spec\":{\"upstream\":\"${POLICY_ENGINE_GRAPH_URI}\"}}"
+----
++
+. Patch the CVO cluster version to use the local OpenShift Update Service:
```

Comment:
s/CVO cluster version/ CVO/  as `CVO cluster version` is confusing.

Reply:
Resolved